### PR TITLE
split up test and create fixtures for ffi_files and cube_file

### PR DESCRIPTION
**Pull request by falkben**
in an effort to debug a subset of a large test that is currently failing I created a few fixtures and split up the test

i am marking the failing test w/ xfail currently, instead of commenting it out. next step will be trying to address the small cutouts test which is xfailed

### CODE REVIEW

File: astrocut/tests/test_cube_cut.py [hunk 1]
```diff
@@ -231,71 +243,119 @@ def test_cutout_extras(tmp_path, ffi_type):
...
 
-    #############################
-    # Test _get_full_cutout_wcs #
-    #############################
-    cutout_size = [5, 3]
+    assert (xmax - xmin) == 8
+    assert (ymax - ymin) == 15
+
+
+# todo: figure out what this is testing and re-enable
```
Comment (jaymedina) [inline astrocut/tests/test_cube_cut.py:288]: Should remove this `todo` comment

Comment (falkben) [reply astrocut/tests/test_cube_cut.py:288]: yup, that's the plan, once i figure out what the problem is with it!

Comment (jaymedina) [reply astrocut/tests/test_cube_cut.py:288]: Oh, I thought it was uncommented because it works now

Comment (falkben) [reply astrocut/tests/test_cube_cut.py:288]: no, it still does not work. it's marked with xfail

Comment (jaymedina) [reply astrocut/tests/test_cube_cut.py:288]: ah yep, see that now. thanks

**Pull request merged**